src/modules/flow/string-regexp/Kconfig

```diff
@@ -0,0 +1,15 @@
+config FLOW_NODE_TYPE_STRING_REGEXP
+	tristate "Node type: string-regexp"
```

Comment:
regulare expressions are only meaningful in the sense of string, so there is no need to call it "string-regexp", just "regexp".